snuba/web/query.py

```diff
@@ -172,12 +172,14 @@ def _format_storage_query_and_run(
     ) as span:
         span.set_tag("table", source)
         try:
-            span.set_tag(
-                "ast_query",
-                AstSqlQuery(clickhouse_query, request_settings).format_sql(),
-            )
+            sql_data = AstSqlQuery(clickhouse_query, request_settings).sql_data()
+            span.set_tag("query_type", "ast")
```

Comment:
Could you please add both the ast query and the. dict query to the span? The ast query is built along the old representation (the dict one) but. it is not used for query execution yet, we are still working on it, so we need both to be present in the span for comparison.